config/internal_users.yml

```diff
@@ -11,7 +11,7 @@ _meta:
 ## Demo users
 
 admin:
```

Comment:
What about rather than using/reusing this user, we dynamically make a special user that has the `admin` backend role using a supplied password? (maybe even a supplied username, but I don't want to feature creep too much)?  Then we can delete this admin entry from the internal user configuration entirely.

This doesn't prevent a cluster from having an internal user config applied with `admin:admin` declared in it.  It also means that when saving the security configuration from the cluster, you'll get that new user with their password in the file too.

Reply:
I am still working on how to use it. It is not used right now because I have not found where I can populate it and guarantee the configurations are live before it tries to access them. 

It is in draft because I am not quite ready for review--not that I don't appreciate you looking--I just am still working on it and wanted to be able to more easily review changes for my own work then what IntelliJ offers for the git diffs.

Reply:
It is gonna have to be the normal internal user objects since we cannot create a persistent version of the `User` class.